index.html

```diff
@@ -0,0 +1,150 @@
+                        <h3>Home</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/magnifying-glass.png" alt="magnifying-glass">
+                        <h3>Explore</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/bell.png" alt="bell">
+                        <h3>Notifications</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/email.png" alt="email">
+                        <h3>Messages</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/bookmark.png" alt="bookmark">
+                        <h3>Bookmarks</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/account.png" alt="account">
+                        <h3>Profile</h3>
+                    </li>
+                    <li class="two-columns align-center">
+                        <img class="gap-sm" src="icons/more.png" alt="more">
+                        <h3>More</h3>
+                    </li>
+                    <button class="gap-sm left-btn">Tweet</button>
+                </ul>
+            </div>
+
+            <div class="left-box three-columns align-center">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p class="text-name">Anica Rahman<br>
+                    @Solo Ann</p>
+                    <img class="more-icon" src="icons/more.png" alt="more">
+                </div>
+            </div>
+
+            <!--MIDDLE SECTION LAYOUT-->
+        <div class="tweet-feed">
+            <div class="middle-box">
+                <h2>Home</h2>
+            </div>
+            <div class="middle-box">
+                <div class="post two-columns">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p class="text">What's happening?</p>
+                </div>
+                <div class="post seven-columns">
+                    <div><img src="icons/image.png" alt="image"></div>
+                    <div><img src="icons/gif.png" alt="gif"></div>
+                    <div><img src="icons/statistics.png" alt="statistics"></div>
+                    <div><img src="icons/happy.png" alt="happy"></div>
+                    <div><img src="icons/calendar.png" alt="calendar"></div>
+                    <div class="space"></div>
+                    <button>Tweet</button>
+                </div>
+            </div>
+            <div class="middle-box">
+                <div class="post two-columns">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
```

Comment:
instead of using the strong tag in html you could of customized it inside of css